lib/app/console_app.rb

```diff
@@ -236,37 +236,39 @@ def read_configurable_fields
       end
     end
 
```

Comment:
This file still has the initialization of logger and TZ on top:

https://github.com/elastic/connectors-ruby/blob/ae4750a59d25889fb20153aed792a75038ea93a1/lib/app/console_app.rb#L21-23

Should we delete it here too?